scripts/install_environment.sh

```diff
@@ -150,6 +150,25 @@ EOF
   sudo chmod +x $fname
 }
 
+function config_nginx() {
+  echo "Config nginx"
+  sudo dnf install -y nginx
+  fname=/etc/nginx/ntp.conf
+  sudo cat <<EOF > $fname
+stream {
+	server {
+		listen [::]:123 udp;
+		proxy_pass clock.redhat.com:123;
```

Comment:
Do I understand correctly that you always forward NTP requests to clock.redhat.com? If so, it might not work on some machines. IMO this should be a variable. Better yet, run a local NTP server and use it as an additional NTP source.

Reply:
Regarding clock.redhat.com, maybe ask @YuviGold .  We do it today for IPv4.  Here we just convert to IPv6.